consensus/state_sync/module.go

```diff
@@ -34,9 +25,14 @@ type StateSyncModule interface {
 	HandleGetBlockResponse(*typesCons.GetBlockResponse) error
 
 	IsServerModEnabled() bool
-	EnableServerMode()
+	EnableServerMode() error
+	DisableServerMode() error
 
 	SendStateSyncMessage(msg *typesCons.StateSyncMessage, nodeAddress cryptoPocket.Address, height uint64) error
+	IsOutOfSync() bool
+	AggregateMetadataResponses() error
```

Comment:
I kind of imagined this to be an independent process that happens implicitly. Anytime you get a response from others, you auto-aggregate.

Reply:
That is right, I left it there to clarify the process for the next steps, removed with the new structure.